templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/template_schema.json

```diff
@@ -16,8 +16,7 @@
       "title": "Linux image",
       "description": "Select Linux image to use for VM",
       "enum": [
-        "Ubuntu 18.04",
-        "Ubuntu 18.04 Data Science VM"
+        "Ubuntu 22.04 LTS"
```

Comment:
This is updated just in time for Ubuntu's 24.04 release ;)

Reply:
#3925